app/assets/v2/css/home.css

```diff
@@ -444,3 +559,111 @@ figure:hover::before {
   border-bottom: 4px solid var(--gc-green);
   font-weight: 700;
 }
+
+@media screen and (max-width: 780px){
+  .banner {
+    background-size: cover;
+  }
+
+  .banner__h2 {
+    font-size: 1.4rem;
+    line-height: 2rem !important;
+    margin-bottom: 1.5rem;
+    margin-left: 2rem;
+    margin-right: 2rem;
+  }
+
+  .section-title {
+    font-size: 1.5rem;
+    border-bottom: 0.375rem solid #121315;
+  }
+  .how-it-works img {
+    margin: 0 auto !important;
+  }
+
+  .hide_on_mobile {
+    display: none;
+  }
+  .show_on_mobile {
+    display: block;
+  }
+  .dark-section .section-title {
+    border-bottom: 0.375rem solid white;
+  }
+
+  .dark-section {
+    padding: 0 !important;
+  }
+  .tribe-points {
+    list-style: none;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 3rem 1rem;
+    text-align: left;
+    padding-left: 0 !important;
+    margin-top: 0 !important;
+  }
+
+  .tribe-points li {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .tribe-points img {
+    height: 2rem;
+  }
+  .tribe-points h4 {
+    font-size: 1.2rem;
+  }
+  .tribe-points p {
+    font-size: 0.875rem;
+  }
+
+  .tribes {
+    width: 100% !important;
+  }
+  .tribes .justify-content-center {
+    margin-top: 0!important;
+    margin-bottom: 0!important;
+  }
+  .tribes .tribes-grid {
+    display: grid !important;
+    grid-gap: 16px;
+    grid-template-columns: repeat(2, 1fr);
+  }
+  .testimonial-card {
+    margin-left: 5%;
+    margin-right: 5%;
+    height: 300px;
+  }
+  .testimonial-card:first-child {
+    margin-left: -60%;
+  }
+  .testimonial-card:last-child {
+    margin-right: -60%;
+  }
+  .card-description {
+    width: 80%;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+  }
+  .card-description::before {
+    font-size: 32px;
+  }
+  .card-description::after {
+    font-size: 32px;
+
+  }
+  .overlap-line {
+    margin-top: -30px;
+  }
+  .avatar {
+    max-width: 60px;
+    max-height: 60px;
+  }
+  .press-mobile {
+    padding-left: 35px;
+    padding-right: 35px;
+  }
+}
```

Comment:
^ Add an empty  line after every rule declaration 
Let's reuse use font classes we already have + bootstrap classes